contracts/example/src/event.rs

```diff
@@ -1,17 +1,36 @@
-use soroban_sdk::{Bytes, Env, String, Symbol};
+extern crate std;
```

Comment:
```suggestion
#[cfg(any(test, feature = "testutils"))]
extern crate std;
```
feature gate it since `std` is only needed for `EventTestutils` implementation

Reply:
Completely removed this given self-reference fix